gpu4pyscf/gto/moleintor.py

```diff
@@ -265,21 +245,18 @@ def get_int3c1e(mol, grids, direct_scf_tol):
 
         row, col = np.tril_indices(nao)
         int3c_grid_slice[:, row, col] = int3c_grid_slice[:, col, row]
-        ao_idx = np.argsort(intopt.ao_idx)
+        ao_idx = np.argsort(intopt._ao_idx)
         grid_idx = np.arange(ngrids_of_split)
         int3c_grid_slice = int3c_grid_slice[np.ix_(grid_idx, ao_idx, ao_idx)]
 
         int3c_grid_slice.get(out = int3c[i_grid_split : i_grid_split + ngrids_of_split, :, :])
 
     return int3c
 
-def get_int3c1e_charge_contracted(mol, grids, charges, direct_scf_tol):
+def get_int3c1e_charge_contracted(mol, grids, charges, intopt):
```

Comment:
Set intopt as optional argument.

Reply:
This is an internal function, the interface function https://github.com/pyscf/gpu4pyscf/blob/03d3044e8abb53bd8eac2374fcb91231a0d03b96/gpu4pyscf/gto/moleintor.py#L415
has intopt as optional. Should I use underscore prefix for this function?

Reply:
OK. This should be enough.